src/canmatrix/formats/fibex.py

```diff
@@ -101,7 +101,7 @@ def dump(db, f, **options):
     cluster.set('ID', 'canCluster1')
     create_short_name_desc(cluster, "clusterShort", "clusterDesc")
     create_sub_element_fx(cluster, "SPEED", "500")
-    create_sub_element_fx(cluster, "IS-HIGH-LOW-BIT-ORDER", "false")
```

Comment:
hi  @rawiamoalla 

are you sure here?  I did not double check, so just a question.

This is a change - the rest of your patch is no problem.

Reply:
Hi @ebroecker normally yes. there is no problem if we replaced to true

Reply:
ok, perfect